### DISCUSSION TITLE
Configuration Reload Fails on Mac

### DISCUSSION THREAD

**Discussion by assemble-code** [ghostty-org/ghostty#3772] [Issue Triage] [answered]
Discussion Number:
Subsequent Configuration Reload Fails on Mac Machine.

Steps

    Make an invalid key value pair entry in the config file
    Reload the terminal and we get an error, if we click on ignore then any changes we do after that to the config file is never reloaded again.

I tried restarting the Mac, completely killing the Terminal but nothing works.

**Comment by mitchellh** [comment, marked as answer]
I'm unable to reproduce this on tip. A lot has changed in the past 9 months, so I suspect this got fixed along the way, or your reproduction just isn't working for me. If this issue is still happening for you or others please open a new discussion with reproduction steps. Thank you!
